shared/constants/tokens.js

```diff
@@ -39,3 +40,8 @@ export const STATIC_MAINNET_TOKEN_LIST = Object.keys(contractMap).reduce(
 
 export const TOKEN_API_METASWAP_CODEFI_URL =
   'https://token-api.metaswap.codefi.network/tokens/';
+export const MAX_TOKEN_ALLOWANCE_AMOUNT = new BigNumber(2)
+  .pow(256)
+  .minus(1)
+  .toString(10);
+export const regex = /^[0-9]{1,}([,.][0-9]{1,})?$/u;
```

Comment:
regex is not a descriptive name for this, and also the standard is upper case for constants

Reply:
@brad-decker You are right, sorry for the poor name. Do you know maybe how can I call this? Maybe INPUT_VALUE_REGEX or TOKEN_ALLOWANCE_VALUE_REGEX?

Reply:
In my [latest commit](https://github.com/MetaMask/metamask-extension/pull/16676/commits/1aef7377e096dc77adfbdc45f031da6c00477d00) I renamed that constant to `TOKEN_ALLOWANCE_VALUE_REGEX`. If you have some better suggestion please comment and I will rename it again. Thank you in advance @brad-decker!